src/hud.js

```diff
@@ -815,6 +824,28 @@ export class HUD extends Application {
             })
         })
 
+        html.find('[data-action=toggle-attitude]').on('click', event => {
+            const { attitude } = getAttitude(actor)
+
+            const content = [
+                { value: 'helpful', label: game.i18n.localize(ATTITUDES.helpful.label) },
+                { value: 'friendly', label: game.i18n.localize(ATTITUDES.friendly.label) },
+                { value: 'indifferent', label: game.i18n.localize(ATTITUDES.indifferent.label) },
+                { value: 'unfriendly', label: game.i18n.localize(ATTITUDES.unfriendly.label) },
+                { value: 'hostile', label: game.i18n.localize(ATTITUDES.hostile.label) },
+            ]
+
+            this.#createHUDLockedListTooltip({
+                content,
+                event,
+                selected: attitude,
+                onClick: value => {
+                    if (value === 'default') actor.update({ 'flags.pf2e-hud.attitude': null })
```

Comment:
shouldn't this be `indifferent` instead of `default`, you don't have a `default` value

Reply:
Yeah, it's fixed locally. Never pushed it to the pr because you closed it 👍